src/annotation.js

```diff
@@ -21,6 +22,11 @@ export class Annotation extends Modifier {
     return 'annotations';
   }
 
+  // Is there a better source for this?
+  static get distanceBetweenLines() {
```

Comment:
I'm surprised that there's no global constant here, I thought it was in `tables.js` or `stave.js`.

I suggest adding `STAVE_LINE_DISTANCE` into the top at `tables.js` and using that here, and also please update `stave.js` and maybe `tabstave.js` to use that constant.

Reply:
tabstave currently has `spacing_between_lines_px: 13`, you still want me to change it?

Reply:
Updated stave and annotation to use new constant.  I think tabstave change would break those test cases.